docs/new_instance.md

```diff
@@ -4,7 +4,8 @@ We don't yet have a scriptworker provisioner, so spinning up new instances of a
 
 ## signing scriptworker
 ### gpg keypair
-Generate and sign a gpg keypair for cltsign@**fqdn**, per [these docs](chain_of_trust#adding-new-worker-gpg-keys).
+Generate and sign a gpg keypair for cltsign@**fqdn**, per [these docs](chain_of_trust.html#adding-new-worker-gpg-keys).
+http://scriptworker.readthedocs.io/en/latest/chain_of_trust.html#adding-new-worker-gpg-keys
```

Comment:
I guess the raw link here is not necessary, and will be generated via the markdown link above. Is that correct?